pkg/ibmvpc/explainability_test.go

```diff
@@ -28,544 +28,300 @@ func getConfig(t *testing.T, fileName string) *vpcmodel.VPCConfig {
 	return nil
 }
 
-type explainGeneralTest struct {
-	name        string // test name
-	inputConfig string // name (relative path) of input config file (json)
-	ESrc        string
-	EDst        string
-	EProtocol   common.ProtocolStr
-	ESrcMinPort int64
-	ESrcMaxPort int64
-	EDstMinPort int64
-	EDstMaxPort int64
-	out         string
-}
-
-var explainTests = []*explainGeneralTest{
+var explainTests = []*vpcGeneralTest{
 	{
 		name:        "VsiToVsi1",
-		inputConfig: "input_sg_testing1_new",
+		inputConfig: "input_sg_testing1_new.json",
```

Comment:
Why is the suffix `*json` required

Reply:
because we get the input file from here

Reply:
What I meant is why can't we do without the suffix and add it automatically as in `analysis_output_test.go`

Reply:
I also think we should try to be consistent